lib/gemfury/command/app.rb

```diff
@@ -48,7 +48,7 @@ def versions(gem_name)
       va = []
       va = [ %w{ version uploaded uploaded_by } ]
```

Comment:
If `time_ago` returns variable length (see comment below), maybe the `uploaded_at` column should be last here.